ricecooker/classes/nodes.py

```diff
@@ -9,14 +9,21 @@
 from le_utils.constants import format_presets
 from le_utils.constants import languages
 from le_utils.constants import roles
-
+from le_utils.constants.labels import levels, resource_type, accessibility_categories
+from le_utils.constants.labels import learning_activities, needs, subjects
 from .. import __version__
 from .. import config
 from ..exceptions import InvalidNodeException
 from .licenses import License
 
 MASTERY_MODELS = [id for id, name in exercises.MASTERY_MODELS]
 ROLES = [id for id, name in roles.choices]
+GRADE_LEVELS = [id for id, name in levels.choices]
+RESOURCE_TYPES = [id for id, name in resource_type.choices]
+ACCESSIBILITY_CATEGORIES = [id for id, name in accessibility_categories.choices]
```

Comment:
Each of these label types has an explicit list of the available labels predefined in the le-utils module: https://github.com/learningequality/le-utils/blob/main/le_utils/constants/labels/accessibility_categories.py#L23 so can avoid doing these list comprehensions for these.

Reply:
Changed.